test/unit/storage/FileResourceStore.test.ts

```diff
@@ -429,11 +429,11 @@ describe('A FileResourceStore', (): void => {
   it('errors when mapping a filepath that does not match the rootFilepath of the store.', async(): Promise<void> => {
     expect((): any => {
       // eslint-disable-next-line dot-notation
-      store['mapFilepathToUrl']('http://wrong.com/wrong');
+      store.resourceMapper['mapFilePathToUrl']('http://wrong.com/wrong');
```

Comment:
What's up with the weird notation here? I think this test should just move to resource mapper now.

Reply:
This was probably a way to get around the fact that that function was private, so should change.